docs/ImageInput.md

```diff
@@ -60,7 +60,6 @@ The `<ImageInput>` component accepts the usual `className` prop. You can also ov
 | Rule name                       | Description                                         |
 |---------------------------------|-----------------------------------------------------|
 | `& .RaImageInput-dropZone`      | Styles pass to the underlying `FileInput` component |
-| `& .RaImageInput-preview`       | Styles pass to the underlying `FileInput` component |
 | `& .RaImageInput-removeButton`  | Styles pass to the underlying `FileInput` component |
```

Comment:
Just realized that all the other class names given here are wrong, since `ImageInput` actually uses `FileInputClasses`.
IMO it should have its own classes object (and hence the doc would be correct), but this would be a BC...
So I don't know what's better here. Probably just to fix the rest of the docs in the end...
What do you think?

Reply:
Not very fond of this last option, I would find it confusing to mix classname prefixes for a single component.
For now, I'd recommend only updating the docs.